awkward-cpp/src/cpu-kernels/awkward_BitMaskedArray_to_IndexedOptionArray.cpp

```diff
@@ -1,148 +1,130 @@
 // BSD 3-Clause License; see https://github.com/scikit-hep/awkward/blob/main/LICENSE
 
-#define FILENAME(line) FILENAME_FOR_EXCEPTIONS_C("src/cpu-kernels/awkward_BitMaskedArray_to_IndexedOptionArray.cpp", line)
+#define FILENAME(line)                                                    \
+  FILENAME_FOR_EXCEPTIONS_C(                                              \
+      "src/cpu-kernels/awkward_BitMaskedArray_to_IndexedOptionArray.cpp", \
+      line)
 
 #include "awkward/kernels.h"
 
 template <typename T>
-ERROR awkward_BitMaskedArray_to_IndexedOptionArray(
-  T* toindex,
-  const uint8_t* frombitmask,
-  int64_t bitmasklength,
-  bool validwhen,
-  bool lsb_order) {
+ERROR
+awkward_BitMaskedArray_to_IndexedOptionArray(T* toindex,
+                                             const uint8_t* frombitmask,
+                                             int64_t bitmasklength,
+                                             bool validwhen,
+                                             bool lsb_order) {
```

Comment:
This kind of indentation is bad for limited horizontal space. Also, there's a reason black chose to put all of the arguments on a different line from the function name: with this formatting, changing the length of the name would change the indentation of all of the arguments, which is a bigger, more noisy change.

If it's possible to ask for black-style indentation here, that would be an improvement.